docs/users/author-guide.md

```diff
@@ -1,28 +1,106 @@
+
+Vega files can be included in the ComBINE/OMEX archive of a simulation project to enable visualization on the [project view page](./user-guide.md#visualizations).
+
+More information on how to create Vega files that can incorporate data from BioSimulations projects can be found at [here](../concepts/Visualizations.md).
+
+### Metadata 
+BioSimulations uses the [OMEX-Meta](https://co.mbine.org/standards/omex-metadata) specification to capture metadata about the simulation project. BioSimulations has a list of recommended metadata fields that are used to display the [metadata](./user-guide.md#metadata) of a simulation project. Information about how to use theses fields can be found [here](../concepts/Metadata.md).
 
 ### COMBINE/OMEX archives
 
+BioSimulations uses COMBINE/OMEX archives to bundle the multiple files typically involved in modeling projects into a single archive. These files include, but are not limited to:
+
+- Models (e.g., in CellML, SBML format)
+- Simulation experiments (SED-ML files)s
+- Visualizations for visualizing simulation results (e.g., in Vega format)
+- Supplementary files, such as data used to calibrate the model
+- Metadata about the simulation project (RDF files that follow the OMEX Metadata guidelines)
+ 
 ## Validating projects
 
+BioSimulations provides a validation tool for validating the contents of a COMBINE/OMEX archive. This tool is available on [runBioSimulations](https://run.biosimulations.org/utils/validate-project). The user can upload a COMBINE/OMEX archive or provide a URL to a COMBINE/OMEX archive. The tool will validate the contents of the archive and report any errors. The interface provides a customizable report of the validation results, including validation of the model file, Sed-ML file, metadata, archive manifest, and any images presented in the archive.
+
+![validation-tool](../images/validate.png)
+
+!!!warning
+    This application can only validate targets for simulation observables for unchanged models. Targets for modified models cannot be validated statically, independently from executing the associated SED-ML document. Simulation tools validate such targets when they execute their parent SED-ML document. For the same reason, this application cannot validate targets for model changes. Simulation tools validate such model change targets during their execution.
+
+
+
+This validation can also be accessed programmatically from the [BioSimulations API](https://combine.api.biosimulations.org).
+
+
+
+
 ## Using BioSimulators to test projects prior to submission to BioSimulations
 
+Authors can run their simulation projects locally using the BioSimulators docker images. To run the project locally, pull the appropriate BioSimulators docker image, and then run the simulator as follows:
```

Comment:
"docker" is spelled "Docker"

Reply:
Before this I think its helpful to communicate that BioSimulations uses BioSimulators to execute simulations. Together with this paragraph, this communicates that authors can execute simulations just as BioSimulations does, using the same interfaces to the same tools.